File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
import java.util.*;
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        boolean[] check = new boolean[10];
        
        for(int num : numbers)
        {
            check[num] = true;
        }
        System.out.println(Arrays.toString(check));
        for(int i=0;i<10;i++)
        {
            
            if(!check[i])
            {
                System.out.println(i);
                answer +=i;
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.js
function solution(numbers) {
    var answer = 0;
    
    const arr = Array(10).fill(false);
    
    numbers.forEach((v) => arr[v] = true);
    console.log(arr);
    for(let i =0;i<10;i++)
        {
            if(!arr[i]) answer +=i;
        }
    
    return answer;
}